core/dovecot/Dockerfile

```diff
@@ -13,9 +24,11 @@ RUN pip3 install "podop>0.2.5"
 
 # Image specific layers under this line
 RUN apk add --no-cache \
-  dovecot dovecot-lmtpd dovecot-pop3d dovecot-submissiond dovecot-pigeonhole-plugin rspamd-client \
+  dovecot dovecot-lmtpd dovecot-pop3d dovecot-submissiond dovecot-pigeonhole-plugin rspamd-client xapian-core \
   && mkdir /var/lib/dovecot
 
+COPY --from=0 /usr/lib/dovecot/lib21_fts_xapian_plugin.* /usr/lib/dovecot/
```

Comment:
I'm not sure which way is best practice but at least I think it's clearer to name the build-stage container and do `--from=buildcontainername`, If you agree could you change that? If you disagree it's fine too :D

Reply:
Thanks for the hint and sorry for the late reply. That is indeed much clearer, and I wasn’t aware yet.